tickets/src/main/java/no/unit/nva/publication/ticket/create/TicketResolver.java

```diff
@@ -112,13 +122,18 @@ private TicketEntry handleCreationException(Exception exception, TicketEntry new
         throw new RuntimeException(exception);
     }
 
-    private void publishPublication(Publication publication) {
+    private void publishPublicationAndFiles(Publication publication) {
         var updatedPublication = toPublicationWithPublishedFiles(publication);
         attempt(() -> resourceService.updatePublication(updatedPublication));
         attempt(() -> resourceService.publishPublication(UserInstance.fromPublication(updatedPublication),
                                                          updatedPublication.getIdentifier()));
     }
 
+    private void publishMetadata(Publication publication) {
+        attempt(() -> resourceService.publishPublication(UserInstance.fromPublication(publication),
```

Comment:
Extract duplicate code in publishPublicationAndFiles and publishMetadata?

Reply:
Da må jeg nesten flytte alt både publish og update opp ett nivå, som gjør at resolvePublishingRequest() blir for stor? Har refaktorert litt, ta en titt :)

Reply:
Jeg mente slik:
```
    private void publishPublicationAndFiles(Publication publication) {
        var updatedPublication = toPublicationWithPublishedFiles(publication);
        attempt(() -> resourceService.updatePublication(updatedPublication));
        publishPublication(updatedPublication);
    }

    private void publishMetadata(Publication publication) {
        publishPublication(publication);
    }

    private void publishPublication(Publication updatedPublication) {
        attempt(() -> resourceService.publishPublication(UserInstance.fromPublication(updatedPublication),
                updatedPublication.getIdentifier()));
    }
```
Men dette er pirk, du kan overse

Reply:
Det ja, da er det gjort